tests/handlers/tests.py

```diff
@@ -38,7 +38,8 @@ def test_bad_path_info(self):
         environ['PATH_INFO'] = b'\xed' if six.PY2 else '\xed'
         handler = WSGIHandler()
         response = handler(environ, lambda *a, **k: None)
-        self.assertEqual(response.status_code, 400)
+        # The path of the request will be encoded to '/%ED'.
```

Comment:
I'm not sure if this remains an appropriate test if it doesn't hit this block anymore? 
https://github.com/django/django/blob/d4eefc7e2af0d93283ed1c03e0af0a482982b6f0/django/core/handlers/wsgi.py#L161-L168